### PR TITLE
#101/contributing doc

### LINKED ISSUE #101: Add CONTRIBUTING.md
# Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
<hr>

# Expected Behaviour

Documentation needs to provide guidance for new contributors on how to submit code onto the repo.

CONTRIBUTING.md should provide basic guidance and link the reader to all other documentation.

# Current Behaviour

No documentation beyond README.md 😞

### PR DESCRIPTION
**Pull request by camelPhonso**
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review on my code
- [x] I have made corresponding changes to the documentation (if any were needed)
- [x] My changes generate no new warnings or errors
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and previously existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

# Description

**Closes #101**
Added documentation providing guidance for new developers looking to contribute to Kindly.

I mention some links within the CONTRIBUTING.md doc that are not actually set in this PR:
- Link to Discord needs to be added, if that is infact where we want to send people for questions
- Link to BEST_PRACTICES.md needs to be added (the file has not been created yet and is in another issue)
- Link to installation guide needs to be added

**We originally defined an issue to create a file called BEFORE_YOUR_FIRST_PR.md, but I think now we should maybe make that an installation guide instead** ❓❓ 

### Files changed

`CONTRIBUTING.md` added

### UI changes

No changes to UI ⛔ 

### Changes to Documentation

This PR is for Documentation - only one Markdow file was added.

# Tests

No Changes to behaviour so no new tests. All prior tests are passing

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Help with a question
+
+The `Issues` tab is a working tool for the active development of Kindly, so it should be kept clear of communication not relating to development.
+
+If you have a question related to Trafalgar Girls, Kindly or the community at large:
+
+- Check the documentation first - there's a chance your doubt can be answered by reading the [README](../README.md) file
+- Reach out on [Discord](link to join the server)
+
```
Comment (nichgalzin) [inline .github/CONTRIBUTING.md:48]: Here's the join link for the discord server
https://discord.gg/Chkx4GeZ

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+- Update the status of that Issue to `Blocked`. Other developers will be monitoring Issues with this status and prioritising them in their contributions.
+- Add a comment to the Issue explaining the help you need
+
+## Help with a question
+
+The `Issues` tab is a working tool for the active development of Kindly, so it should be kept clear of communication not relating to development.
+
+If you have a question related to Trafalgar Girls, Kindly or the community at large:
+
+- Check the documentation first - there's a chance your doubt can be answered by reading the [README](../README.md) file
```
Comment (nichgalzin) [inline .github/CONTRIBUTING.md:46]: Not sure about the wording: 'about your doubt'. Sounds a bit strange in English

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Developers at all levels of experience and comfort in the Kindly tech stack are welcome to contribute!
+
+However, before you set out to submit code, please review our [best practices](link to BEST_PRACTICE.md) and our [installation guide](link to INSTALATION.md).
+
+## Finding an Issue
+
+### As a newcomer
+
+If it's your first time submitting code to Kindly and/or you are looking to get acquainted with anyone part of our tech stack or code base, we recommending finding an Issue labeled `Good first issue`. You can filter for these.
```
Comment (nichgalzin) [inline .github/CONTRIBUTING.md:61]: type 'anyone'  to 'any one'

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+### As a senior
+
+If you've been contributing to Kindly for a while (Thanks ! ✨) and are quite comfortable with the tech stack and code base:
+
```
Comment (nichgalzin) [inline .github/CONTRIBUTING.md:66]: Space after 'Thanks' ?

Comment (camelPhonso) [reply .github/CONTRIBUTING.md:66]: no 😶‍🌫️

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (camelPhonso) [reply .github/CONTRIBUTING.md:46]: I don't know what you're on about, I talk English good ❗ (changed it to 'question')

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (camelPhonso) [reply .github/CONTRIBUTING.md]: Fixed in new commit

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (camelPhonso) [reply .github/CONTRIBUTING.md:49]: Added that in now 🔗

**Review by eliazzo — APPROVED** [review]

**Pull request merged**